code/src/components/app-range-slider/utils/check-within-range/checkWithinRange.test.ts

```diff
@@ -0,0 +1,20 @@
+import checkWithinRangeSliderRange from "@/components/app-range-slider/utils/check-within-range/checkWithinRange";
+
+const testCases = [
+  [10, 0, 20, true],
+  [0, 0, 20, true],
+  [20, 0, 20, true],
+  [-1, 0, 20, false],
+  [21, 0, 20, false],
+  [0, 0, 20, true],
+  [20, 0, 20, true]
+] as const;
+
+describe("checkWithinRange", () => {
+  test.each(testCases)(
+    "returns %s when value is %i, min is %i, and max is %i",
+    (value, min, max, expected) => {
+      expect(checkWithinRangeSliderRange({ value, min, max })).toBe(expected);
+    }
```

Comment:
I guess should to give better naming for `%s` `%i` etc. But for now it's not understandable what it means
maybe to use your naming with arguments ? like `value, min, max, expected` ?

Reply:
Sorry, I can't rename it because it is built in placeholders for jest's test.each. You can read more about it in [docs](https://jestjs.io/docs/api#testeachtablename-fn-timeout):

![image](https://github.com/user-attachments/assets/a1e543ef-0818-49d2-b599-d4c5ec25f0f4)